.travis.yml

```diff
@@ -8,12 +8,10 @@ node_js:
 env:
   matrix:
   - TEST_BROWSER_NAME=PhantomJS
-  - TEST_BROWSER_NAME=chrome TEST_BROWSER_OS='Windows 7,Linux'
-  - TEST_BROWSER_NAME=firefox TEST_BROWSER_OS='Windows 7,Linux' TEST_BROWSER_VERSION='30,26'
-  - TEST_BROWSER_NAME='internet explorer' TEST_BROWSER_OS='Windows 7' TEST_BROWSER_VERSION=10
-  - TEST_BROWSER_NAME='internet explorer' TEST_BROWSER_OS='Windows 8.1' TEST_BROWSER_VERSION=11
-  - TEST_BROWSER_NAME=safari TEST_BROWSER_OS='OS X 10.8' TEST_BROWSER_VERSION=6
-  - TEST_BROWSER_NAME=safari TEST_BROWSER_OS='OS X 10.9' TEST_BROWSER_VERSION=7
+  - TEST_BROWSER_NAME=chrome TEST_BROWSER_OS='Windows 10,Linux,OS X 10.11'
+  - TEST_BROWSER_NAME=firefox TEST_BROWSER_OS='Windows 10,Linux,OS X 10.11'
+  - TEST_BROWSER_NAME='internet explorer' TEST_BROWSER_OS='Windows 10'
+  - TEST_BROWSER_NAME=safari TEST_BROWSER_OS='OS X 10.11'
```

Comment:
I wan't to give @DirectXMan12 a chance to comment on this. But I'm fine with only testing the latest versions of browsers seeing as the update rate has always been very high barring IE. When it comes to operating system versions I've never experienced that it makes much difference when it comes to browsing.